src/Selenium2Library/keywords/_element.py

```diff
@@ -480,7 +480,7 @@ def press_key(self, locator, key):
         """Simulates user pressing key on element identified by `locator`.
 
         `key` is either a single character, or a numerical ASCII code of the key
-        lead by '\\'. In test data, '\\' must be escaped, so use '\\\\'.
+        lead by '\\\\'.
```

Comment:
I did have this conflict. Not sure which wording you prefer.  My thoughts is that if it always has to be escaped there is not point in saying otherwise.